include/service/jobs/plugins/plugin_uninstall_job.js

```diff
@@ -99,6 +99,7 @@ module.exports = function PluginUninstallJobModule(pb) {
      */
     PluginUninstallJob.prototype.getWorkerTasks = function(cb) {
         var self = this;
+        var pluginService = new pb.PluginService();
```

Comment:
Instead of declaring this here.  I would use the instance that is created in prototype constructor.  You should be able to reference an instance of PluginService via ```this.pluginService```